node/service/src/parachains_db/upgrade.rs

```diff
@@ -197,6 +228,17 @@ pub(crate) fn paritydb_version_1_config(path: &Path) -> parity_db::Options {
 	options
 }
 
+/// Database configuration for version 2.
+pub(crate) fn paritydb_version_2_config(path: &Path) -> parity_db::Options {
+	let mut options =
+		parity_db::Options::with_columns(&path, super::columns::v2::NUM_COLUMNS as u8);
+	for i in columns::v1::ORDERED_COL {
```

Comment:
One question that might arise in future is why don't we have `ORDERED_COL` in `v2` module (it might be just a copy of `v1` field in this case).

Reply:
Initially I had done the copy paste, but then realized it makes more sense to only have the actual changes in v2.